target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9_bt_homehub-v5a.dts

```diff
@@ -262,12 +294,20 @@
 	status = "okay";
 	reset-gpios = <&gpio 21 GPIO_ACTIVE_LOW>;
 
-	wifi@168c,002d {
+	wifi@1 {
 		compatible = "pci168c,002d";
-		reg = <0x7000 0 0 0 0>;
```

Comment:
is this `reg` change on purpose? if so it should be documented in the commit description

Reply:
Might be a mistake. I don’t remember if I observed dtc warnings.

Reply:
Removed